pages.id/windows/msiexec.md

```diff
@@ -0,0 +1,20 @@
+# msiexec
+
+> Memasang, memperbarui, memperbaiki, atau menghapus program Windows melalui file MSI dan MSP yang tersedia.
+> Informasi lebih lanjut: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec>.
+
+- Memasang sebuah program melalui file MSI:
```

Comment:
I'm not sure which verb mood are you using, but the example descriptions should be in indicative mood.